service/src/main/kotlin/fi/espoo/evaka/pairing/MobileDeviceQueries.kt

```diff
@@ -0,0 +1,68 @@
+package fi.espoo.evaka.pairing
+
+import fi.espoo.evaka.shared.db.Database
+import fi.espoo.evaka.shared.domain.NotFound
+import org.jdbi.v3.core.kotlin.mapTo
+import java.util.UUID
+
+fun insertDeviceData(tx: Database.Transaction, unitId: UUID): UUID {
+    val defaultName = "Nimeämätön laite"
+
+    // language=sql
+    val employeeInsert =
+        """
+            INSERT INTO employee (first_name, last_name, email, aad_object_id)
+            VALUES (:name, (SELECT name FROM daycare WHERE id = :unitId), null, null)
+            RETURNING id
+        """.trimIndent()
+    val id = tx.createQuery(employeeInsert)
+        .bind("unitId", unitId)
+        .bind("name", defaultName)
+        .mapTo<UUID>()
+        .first()
+
+    // language=sql
+    val deviceInsert =
+        """
+            INSERT INTO mobile_device (id, unit_id, name) 
+            VALUES (:id, :unitId, :name)
+        """.trimIndent()
+    tx.createUpdate(deviceInsert)
+        .bind("id", id)
+        .bind("unitId", unitId)
+        .bind("name", defaultName)
+        .execute()
+
+    return id
+}
+
+fun listDevices(tx: Database.Read, unitId: UUID): List<MobileDevice> {
+    // language=sql
+    val sql = "SELECT * FROM mobile_device WHERE unit_id = :unitId AND deleted = false ORDER BY created"
```

Comment:
I'd much rather order in the frontend closer to where the data is used. A good convention is to use `ORDER BY` only when it's required for correctness, explicitly required somehow in the function signature, or if there's an obvious natural order which should always be used. In this case none of these guidelines apply, because ordering is just a visual decision.